lib/bokken/events/event_admin.ex

```diff
@@ -46,4 +47,20 @@ defmodule Bokken.Events.EventAdmin do
       enrollments_close: nil
     ]
   end
+
+  def send_email(users, email) do
+    users
+    |> List.foldl(
```

Comment:
```suggestion
    |> Enum.reduce(
```

We use ```Enum.reduce``` instead of ```List.foldl``` for working with lists`

Reply:
Thanks :pray: